webapp/src/Controller/API/ContestController.php

```diff
@@ -417,13 +417,25 @@ public function getContestYamlAction(Request $request, string $cid): StreamedRes
         $response->setCallback(function () use ($contest, $penalty_time) {
             echo "name:                     " . $contest->getName() . "\n";
             echo "short-name:               " . $contest->getExternalid() . "\n";
+            echo "activation-time:               " .
+                Utils::absTime($contest->getActivatetime(), true) . "\n";
             echo "start-time:               " .
                 Utils::absTime($contest->getStarttime(), true) . "\n";
             echo "duration:                 " .
                 Utils::relTime($contest->getEndtime() - $contest->getStarttime(), true) . "\n";
             echo "scoreboard-freeze-length: " .
```

Comment:
Slightly unrelated, but we should change this to `scoreboard_freeze_duration`, see https://ccs-specs.icpc.io/draft/contest_api#contests and https://ccs-specs.icpc.io/draft/contest_package#example-yaml-files